recipes/mcroni/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "1.0.1" %}
-{% set sha256 = "584afa62047ab9598b8fd89d0d39fbd265baf402103005d5efbbb71b440812a5" %} 
+{% set version = "1.0.0" %}
```

Comment:
You're reverting this to an earlier version it seems. Is that intentional?

Reply:
Good spot. I will do another release incrementing version and make another PR